ng-dev/pr/merge/pull-request.ts

```diff
@@ -46,6 +47,8 @@ export interface PullRequest {
   baseSha: string;
   /** Git revision range that matches the pull request commits. */
   revisionRange: string;
+
+  validationFailures: PullRequestValidationFailure[];
```

Comment:
Add a comment for this field too, and should we actually do this now? Couldn't we continue asserting directly instead of storing the failures here (in the loadAndValidate function)